tests/translate/translate_remap_profile_2d.py

```diff
@@ -63,7 +63,8 @@ def compute(self, inputs):
         inputs["i1"] = self.grid.global_to_local_x(inputs["i1"] - 1)
         inputs["i2"] = self.grid.global_to_local_x(inputs["i2"] - 1)
         inputs["jslice"] = slice(0, 1)
-
+        if "qs" not in inputs:
+            inputs["qs"] = utils.make_storage_from_shape(self.maxshape)
```

Comment:
Makes perfect sense, since `remap_profile.compute()` expects `qs` as one of the inputs.

What I'm not sure how we'll deal with in the long-run, is that we actually don't want to call `make_storage_from_shape` in any of the compute methods when running the dycore, e.g.
```
    qs = utils.make_storage_from_shape(pe1.shape, origin=(0, 0, 0))
```
in `mapn_tracer.compute()`, but this is not related to this PR.

Reply:
agreed! we need to get rid of those as much as possible. We are reducing some of them by putting more code into stencils, but there will still be some we may need to push up a few levels and pass through, or something.